src/components/common/YourCompany/YourCompany.vue

```diff
@@ -7,16 +7,18 @@
 
     <div class="define-company-header">
       <v-icon color="appDkBlue">mdi-domain</v-icon>
-      <label class="define-company-title">Your Company</label>
+      <label class="define-company-title">Your {{ getResource.entityReference }}</label>
```

Comment:
I would include "Your" in the resource, too.

(In case UX decides to change this to "My Company" -- haha -- or a number of other possibilities.)

Reply:
That 'never' happens 😆

Reply:
Updated 👍

Reply:
Actually, change of heart here. I believe it's simpler to leave the reference as a single resource value and apply the variations where used. In this case 'Your`. 
This allows the resource value a little more reusability and flexibility.

Reply:
As per our original design, we would try to have minimal logic in the components to handle filing/entity type variations (so the logic doesn't get complicated). In this case the logic is straight-forward at the moment so I'm ok with this. (Can always be easily changed later.)